seam/modules/action_attempts.py

```diff
@@ -0,0 +1,67 @@
+from typing import Dict, Optional, Union
+import time
+
+from ..client import SeamHttpClient
+from ..exceptions import SeamActionAttemptFailedError, SeamActionAttemptTimeoutError
+from ..routes.models import ActionAttempt
+
+TIMEOUT = 5.0
+POLLING_INTERVAL = 0.5
+
+
+def get_action_attempt(client: SeamHttpClient, action_attempt_id: str) -> ActionAttempt:
+    res = client.post(
+        "/action_attempts/get", json={"action_attempt_id": action_attempt_id}
+    )
+
+    return ActionAttempt.from_dict(res["action_attempt"])
+
+
+def poll_until_ready(
+    client: SeamHttpClient,
```

Comment:
Instead of accepting `client` param, I tried to use the `ActionAttempts` client instance. However, when I import `from ..routes.action_attempts import ActionAttempts` here, it causes a circular import error again.
![image](https://github.com/seamapi/python-next/assets/84702959/5ceb1ff9-ecda-477b-84e8-090147774cd2)

Reply:
Makes sense. This might be a case where ES modules is more powerful than Python imports. Fortunately, since we have the client now, duplication the GET requests is not a big deal.